components/list/list-item-generic-layout.js

```diff
@@ -384,6 +416,26 @@ class ListItemGenericLayout extends RtlMixin(LitElement) {
 		}
 	}
 
+	_onDropdownClose() {
+		if (!this._tooltipShowing) this.frontRow = false;
+		this._dropdownShowing = true;
```

Comment:
I think this should be setting `false` instead.

Reply:
derp.